doc/source/user_guide/fields_container.rst

```diff
@@ -1,23 +1,22 @@
 .. _ref_user_guide_fields_container:
 
 ===========================
-Fields container and fields
+Field containers and fields
 ===========================
 While DPF uses operators to load and operate on data, it uses field containers
```

Comment:
Here, they even used field container....

Reply:
Hi @PipKat @JennaPaikowsky ,
Concerning this, the official term we use for the class in the code is `a FieldsContainer`.
It is a container with multiple `Field` objects, hence usually referred to as a `fields container` in the documentation.

Reply:
@PProfizi Thanks! I will adjust the spots where it says "Field container" to plural, instead of the other way around then.